bazel/grpc_build_system.bzl

```diff
@@ -282,17 +352,18 @@ def grpc_cc_test(name, srcs = [], deps = [], external_deps = [], args = [], data
         flaky: Whether this test is flaky.
         copts: Add these to the compiler invocation.
         linkstatic: link the binary in static mode
+        exclude_pollers: list of poller names to exclude for this set of tests.
+        test_each_engine: set to False if the test is not sensitive to
```

Comment:
How about calling this parameter `uses_event_engine`?  That seems like a much more intuitive name from the perspective of someone reading a test definition.

Also, I suspect that there are a lot of tests that are not setting this to False that should be.  Just from a quick scan, I suspect that almost everything under test/core *aside* from the end2end tests should be doing this.

Reply:
Done. I used that name in a previous PR, but decided that it was misleading: most tests will certainly _use_ an EventEngine, but many of those tests may not care if they're exercised against every EventEngine implementation. It's not a big problem, though.

> Also, I suspect that there are a lot of tests that are not setting this to False that should be.

Agreed. I caught a handful already in the promise and avl code. We can catch more as we go along. It might make sense to have this set to False by default, but that would conflict with how the poller expansion works today.

Reply:
It's probably a sufficient heuristic at the moment to add `uses_event_engine=False` wherever `uses_polling=False` is set. There will be some exceptions to that, such as the EventEngine conformance test suite, but that may be a good first pass. WDYT?

Reply:
I did a first pass on the ~150 tests where `uses_polling=False`. Most are isolated tests that also do not need to exercise all EventEngines, with only a few exceptions. I also ensured that the `message_compress` tests are not parameterized on `poller x engines`, I don't think they need to be.